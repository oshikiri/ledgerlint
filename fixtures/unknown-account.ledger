2020-03-26 * super market
  Expences:Unknown expence  600 JPY
  Assets:Cash  -600 JPY

2020-03-26 * super market
  Expences:Food  600 JPY
  Assets:Unknown asset
