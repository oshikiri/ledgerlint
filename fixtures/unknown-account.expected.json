{"source":"ledgerlint","file_path":"fixtures/unknown-account.ledger","line_number":2,"range":{"start":{"line":1,"character":0},"end":{"line":1,"character":80}},"level":"WARN","severity":2,"message":"unknown account: Expenses:Unknown expense"}
{"source":"ledgerlint","file_path":"fixtures/unknown-account.ledger","line_number":7,"range":{"start":{"line":6,"character":0},"end":{"line":6,"character":80}},"level":"WARN","severity":2,"message":"unknown account: Assets:Unknown asset"}
